fieldbehavior/fieldbehavior_test.go

```diff
@@ -170,7 +184,7 @@ func TestValidateImmutableFieldsWithMask(t *testing.T) {
 				ExternalReferenceId: "I am immutable!",
 			},
 			UpdateMask: &fieldmaskpb.FieldMask{
-				Paths: []string{""},
+				Paths: []string{},
```

Comment:
The fieldmask in this test case should be completely empty. `""` is also considered a value in most other cases in the aip-go library